commit 8bfbf64c94a8e69edab1e6ee3359bf75a457f19e (refs/changes/79/3879/2)
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2018-04-27T17:50:32+00:00 (1 year, 5 months ago)
    
    MORAY-466 Move "electric-moray" to new Triton origin image
    Reviewed by: Jan Wyszynski <jan.wyszynski@joyent.com>
    Approved by: Jan Wyszynski <jan.wyszynski@joyent.com>

diff --git a/targets.json.in b/targets.json.in
index 40eeff2..7b5da7e 100644
--- a/targets.json.in
+++ b/targets.json.in
@@ -1086,9 +1086,10 @@ cat <<EOF
     "build_platform": "20151126T062538Z",
     "image_name": "manta-electric-moray",
     "image_description": "Manta moray proxy",
-    "image_uuid": "b4bdc598-8939-11e3-bea4-8341f6861379",
+    "// image_uuid": "triton-origin-multiarch-15.4.1@1.0.1",
+    "image_uuid": "04a48d7d-6bb5-4e83-8c3b-e60a99e0f48f",
     "pkgsrc": [
-      "haproxy-1.4.24"
+      "haproxy-1.6.2"
     ],
     "repos": [
         {"url": "git@github.com:joyent/electric-moray.git"}
